app/gather/settings.py

```diff
@@ -69,8 +70,16 @@
     if key.startswith(AETHER_PREFIX)
 ]
 
-# ElasticSearch consumer URL
+TENANCY_HEADER = os.environ.get('TENANCY_HEADER', 'X-Oauth-realm')
+
+# ElasticSearch consumer
+CONSUMERS_CONFIG_FILE = os.environ.get('CONSUMERS_CONFIG_FILE', '/code/conf/consumers.json')
+AUTO_CONFIG_CONSUMERS = os.environ.get('AUTO_CONFIG_CONSUMERS')
+CONSUMER_SETTINGS = []
 ES_CONSUMER_URL = os.environ.get('ES_CONSUMER_URL')
```

Comment:
we could remove this one if we are using the config file.

Reply:
Will create another ticket for this

Reply:
https://jira.ehealthafrica.org/browse/GTH-214